subsys/spm/Kconfig

```diff
@@ -35,6 +35,18 @@ config PM_PARTITION_SIZE_SPM
 	  of this configuration needs to match the requirements set by the
 	  script 'partition_manager.py'. See pm.yml.
 
+config PM_PARTITION_SIZE_SPM_RAM
+	hex "RAM reserved for SPM"
+	# On the nRF9160 the BSDLib needs to be included in the build.
+	# There are scrict requirements to the RAM area reserved for BSDLib.
+	# Specifically it must be 64kB from RAM offset 0x10000.
+	# By setting the SPM RAM to 0x10000 we ensure that BSDLib RAM ends
+	# up in the right place.
```

Comment:
Why is this information only available to those who reads the Kconfig file raw ?
If think this looks like very relevant help text to users configuring through `menuconfig`

Would you mind moving to help section instead (feel free to revise the text.)